### PR TITLE
Remove all framework usage of RelativeMouseDrag

### PR DESCRIPTION
**Pull request by peppy**
Should be moved to osu! project as legacy/non-standard behaviour. See #897.

### CODE REVIEW

**Comment by Tom94** [comment]
I prefer to leave ScrollBar outside of ScrollContainer and to keep the constants static, rather than making it nested again.

File: osu.Framework/Graphics/Containers/ScrollContainer.cs [hunk 1]
```diff
@@ -148,33 +156,40 @@ public bool ScrollbarOverlapsContent
         /// </summary>
         public Container<T> ScrollContent => content;
 
-        private bool dragging;
+        protected virtual bool IsDragging { get; set; }
```
Comment (Tom94) [inline osu.Framework/Graphics/Containers/ScrollContainer.cs]: Is it intentional that the setter isn't private? Seems to me like that allows breakage.

**Comment by peppy** [comment]
If we make it outside of the class, it has to be public. I thought nesting to make it protected was a better solution. Thoughts?

**Comment by Tom94** [comment]
It doesn't have to be public. It can be internal as it was before you moved it in.

**Pull request merged**